static/index.js

```diff
@@ -121,3 +133,21 @@ var setupVmCompatibility = function() {
   if (!vm.Script.createContext)
     vm.Script.createContext = vm.createContext;
 }
+
+var profileStartup = function(cacheDir, loadSettings) {
+  var currentWindow = require('remote').getCurrentWindow();
+  currentWindow.openDevTools();
+  currentWindow.once('devtools-opened', function() {
+    setTimeout(function() {
+      console.profile('startup');
+      try {
+        setupWindow(cacheDir, loadSettings);
+      } catch (error) {
+        handleSetupError(error);
+      } finally {
+        console.profileEnd('startup');
+        console.log("Switch to the Profiles tab to view the startup profile")
+      }
+    }, 100);
```

Comment:
@zcbenz this timeout seemed to be required. Without it the profile didn't actually get created. Do you think this is an okay workaround?